app/helpers/carts_helper.rb

```diff
@@ -22,6 +22,11 @@ def display_response_actions?(cart, user)
     current_linear_approval?(cart, user)
   end
 
+  # Todo: Move this to an NCR specific template?
+  def display_restart?(cart, user)
```

Comment:
_Technically_ it's another method (that has internal state), but I hear ya. Whatever you prefer!